### PR TITLE
bug resolved for adding sequencing files

### PR DESCRIPTION
**Pull request by Snehal1894**
This update resolves the issue of having multiple entries of paths of fastq file.
After uploading the csv file in 'import sequencing file' page, only the new files will be added.

### CODE REVIEW

File: cLIMS/organization/export.py [hunk 1]
```diff
@@ -1256,4 +1255,4 @@ def exportDCIC(request):
             return response
     else:
         messages.error(request, 'Please add the all experiments into biological/technical replicate experiment set!')
-        return HttpResponseRedirect('/detailProject/'+request.session['projectId'])
+        return HttpResponseRedirect('/detailProject/'+request.session['projectId'])
```
Comment (sergpolly) [inline cLIMS/organization/export.py]: there is a newline symbol that got lost during editing
not a big deal or problem at all, but ...
when someone else if reviewing a PR, i.e. some code changes - it is easy to get lost when there a ton of such small changes
We should discuss git-best practices at some point.

File: cLIMS/organization/importSeqFiles.py [hunk 2]
```diff
@@ -49,7 +53,7 @@ def importSeqFiles(request,pk):
             if(filePath==True and v[filePathIndx]!=""):
                 path=v[filePathIndx]
             
-            if(expName!= "" and path!= ""):
+            if(expName!= "" and path!= ""):
```
Comment (sergpolly) [inline cLIMS/organization/importSeqFiles.py]: couple of spaces at the end of the line - again , see 1st comment
we need to cleanup to simplify the code review

File: cLIMS/organization/importSeqFiles.py [hunk 3]
```diff
@@ -70,16 +74,29 @@ def importSeqFiles(request,pk):
...
+    context['dupDict'] = dupDictSorted
     context['orderList'] = orderList
-    project=Project.objects.get(pk=pk)
+
     context['project']=project
     
-    return (request, template_name, context)
\ No newline at end of file
+    return (request, template_name, context)
+
```
Comment (sergpolly) [inline cLIMS/organization/importSeqFiles.py]: same thing - undo unnecessary code edits to simplify review

File: cLIMS/organization/importSeqFiles.py [hunk 4]
```diff
@@ -70,16 +74,29 @@ def importSeqFiles(request,pk):
...
     if " " in runDict:
         del runDict[" "]
     
     runDictSorted=sorted(runDict.items())
+    dupDictSorted=sorted(dupDict.items())
     context['runDict'] = runDictSorted
+    context['dupDict'] = dupDictSorted
     context['orderList'] = orderList
-    project=Project.objects.get(pk=pk)
+
```
Comment (sergpolly) [inline cLIMS/organization/importSeqFiles.py]: remove empty line

File: cLIMS/organization/urls.py [hunk 5]
```diff
@@ -133,5 +133,5 @@
     url(r'^downloadFile/$', views.downloadFile, name='downloadFile'),
     url(r'^moveExperiments/(?P<pk>[0-9]+)/$', MoveExperiments.as_view(), name='moveExperiments'),
     url(r'^exportDistiller/(?P<pk>[0-9]+)/$', ExportDistiller.as_view(), name='exportDistiller'),
-    
+
```
Comment (sergpolly) [inline cLIMS/organization/urls.py]: undo - as there was no real change

File: cLIMS/organization/export.py (hunk 1 above)
Comment (Snehal1894) [reply cLIMS/organization/export.py]: Makes sense.

**Comment by nandankita** [comment]
> Looks great @Snehal1894 ! @nandankita what do you think ?

Changes looks good. Once we test these changes on Friday, will update the code in production.

File: cLIMS/organization/importSeqFiles.py (hunk 4 above)
Comment (sergpolly) [reply cLIMS/organization/importSeqFiles.py]: disregard this particular one

File: cLIMS/organization/urls.py (hunk 5 above)
Comment (sergpolly) [reply cLIMS/organization/urls.py]: looks like 4 spaces to me

**Pull request merged**